docs/source/configuration_file.rst

```diff
@@ -297,7 +297,9 @@ UNet3D (Optional)
    translation of 1 voxel in the 1st dimension, 2 voxels in the 2nd
    dimension and 3 voxels in the 3rd dimension at every iteration until
    the whole input matrix is covered.
--  ``attention_unet``: Bool. Use attention gates in the Unet's decoder.
+-  ``attention_unet`` (optional): Bool. Use attention gates in the Unet's decoder.
+- ``n_filters`` (optional): Int. Number of filters in the first convolution of the UNet. This number of filters will be doubled at each convolution.
+- ``relu`` (optional): Bool. Sets final activation to normalized ReLU (relu between 0 and 1).
```

Comment:
I would have thought `relu` is only needed in the default model, see update [here](https://github.com/ivadomed/ivadomed/blob/master/ivadomed/main.py#L114).

Reply:
Bien vu! updated :)